packages/webpack/template-webpack-plugin/src/LynxTemplatePlugin.ts

```diff
@@ -570,6 +576,45 @@ class LynxTemplatePluginImpl {
         );
       });
 
+      const LynxCacheEventsSetupListRuntimeModule =
```

Comment:
Instead of in `LynxTemplatePlugin`, can we make a new standalone webpack plugin? Then we could test only the plugin itself.

You can apply the plugin inside the `LynxTemplatePlugin` or in react-rsbuild-plugin.